src/main/java/com/kiki/user/UserController.java

```diff
@@ -8,17 +8,20 @@
 import java.time.ZonedDateTime;
 import java.util.Map;
 
+@RequestMapping("/users")
 @RestController
 public class UserController {
     private final UserService userService;
     public UserController(UserService userService) {
         this.userService = userService;
     }
-    @GetMapping("users/{id}")
+
+    @GetMapping("/{id}")
     public User getUser(@PathVariable("id") int id) {
         return userService.findUser(id);
     }
-    @GetMapping("users/{id}/addressId/{addressId}")
+
+    @GetMapping("/{id}/addressId/{addressId}")
```

Comment:
REST APIのパス設計にしたがうとこうなりますね！
```suggestion
    @GetMapping("/{id}/address/{addressId}")
```

Reply:
訂正いたしました。
https://github.com/KIKI0911/Task_7/compare/main...feature_imp#diff-b26cb092ea3f7ce56b5f005b03ee1fd5193281c6e95e3b7ad38f16a6fcce2a36R32